src/app/modules/main/chat_section/module.nim

```diff
@@ -702,7 +702,7 @@ method onCommunityChannelEdited*(self: Module, chat: ChatDto) =
 method switchToOrCreateOneToOneChat*(self: Module, chatId: string) =
   # One To One chat is available only in the `Chat` section
   if(self.controller.getMySectionId() != singletonInstance.userProfile.getPubKey()):
-    return
+    self.delegate.setActiveSectionById(singletonInstance.userProfile.getPubKey())
```

Comment:
IMO, it should be the main module's job to switch section, so we can listen to `SIGNAL_CHAT_SWITCH_TO_OR_CREATE_1_1_CHAT` in `main/controller` and switch section in that module.